CONTRIBUTION.md

```diff
@@ -28,58 +28,63 @@ We know that as of 1.12 this is not the case for great parts of the mod, but we
 Unless your PR is directly related to code cleaning then your PR should not contain any formatting changes to unchanged code blocks as this simply makes reviewing them unnecessarily complex.
 
 ### Formatting
-* Use 4 space indent instead of Tabs
-* operators should be surrounded by spaces
-  * For example `if (statement == true)`
-* a `,` comma to separate types should be followed by exactly one space
-* type casts should be followed by exactly one space
-* `{` should not be prefixed with returns
-* Methods should have exactly one free line between them (or their doc blocks)
-* Avoid empty lines at the start of class, interface or enum definitions
-* Avoid unnecessary lines in empty Methods
-    * `public void emptyImpl() {}`
-    * Also try and avoid introducing code that may require empty methods
-* Method/Class annotations should gain their own lines in front of the definition
-* Parameter Annotations should be surrounded by exactly one space
-* Comments should always start with a single space `// Comment`
+* Use 4 space indent instead of Tabs.
+* Operators should be surrounded by spaces:
+    * For example `if (statement == true)`.
+* A `,` comma to separate types should be followed by exactly one space.
+* Type casts should be followed by exactly one space.
+* `{` should not be wrapped onto a new line.
+* Methods should have exactly one free line between them (or their doc blocks).
+* Avoid empty lines at the start of class, interface or enum definitions.
+* Avoid unnecessary lines in empty Methods:
+    * `public void emptyImpl() {}`.
+    * Try to avoid introducing code that may require empty methods.
+* Method/Class annotations should gain their own lines in front of the definition.
+* Parameter Annotations should be surrounded by exactly one space.
+* Comments should always start with a single space `// Comment`.
+* Import statments should be ordered in sections with exactly 1 empty line between them.
+    1. All other unmatched imports, should be in alphebetical order
+    2. `java.`, `javax.`, etc.
```

Comment:
```suggestion
    2. `java.` & `javax.`
```
Those seem to be the only ones exempt from the main alphabetical ordering above.